src/providers/postgres/CMakeLists.txt

```diff
@@ -40,7 +44,7 @@ INCLUDE_DIRECTORIES(
   ${CMAKE_CURRENT_BINARY_DIR}/../../ui
 )
 
-ADD_LIBRARY (postgresprovider MODULE ${PG_SRCS} ${PG_MOC_SRCS})
+ADD_LIBRARY (postgresprovider MODULE ${PG_SRCS} ${PG_HDRS} ${PG_MOC_SRCS})
```

Comment:
Because it otherwise does not appear in the project file tree in various IDEs, QtCreator for one. It is common practice in CMake (and other build systems for the matter) to add headers also.